articles/cognitive-services/Speech-Service/quickstarts/text-to-speech/async-synthesis-long-form-audio.md

```diff
@@ -234,13 +244,18 @@ if args.delete:
 
 ### Test your code
 
-Run this command, replacing `<your_key>` with your Speech subscription key, and `<region>` with the region where your Speech resource was created (for example: `eastus` or `westus`). This information is available in the **Overview** tab for your resource in the [Azure portal](https://aka.ms/azureportal).
+Let's check your submitted requests before. You'll need to update a few things in the request below:
```

Comment:
This is a bit unclear. Are you trying to say that the following request is used to check what requests had been made previously?

Reply:
yes. This action is to check all the synthesis request under user's Azure subscription.

Reply:
How about this? (see below)